agent.go

```diff
@@ -72,6 +72,7 @@ const (
 	cannotGetTimeMsg   = "Failed to get time for event %s:%v"
 	pciBusRescanFile   = "/sys/bus/pci/rescan"
 	pciBusMode         = 0220
+	cpuMode            = 0644
```

Comment:
I think `0600` should be enough.

Reply:
Yeah, maybe even `0200`?

Reply:
this file is used to set and get the cpu status, ```0600``` should be enough